src/components/standard/article.tsx

```diff
@@ -103,10 +104,13 @@ const Standard = ({ item, children }: Props): JSX.Element => {
                 {children}
             </Body>
             {epicContainer}
-            <footer css={articleWidthStyles}>
+            <section id="tags" css={articleWidthStyles}>
                 <Tags tags={item.tags} format={item}/>
-            </footer>
+            </section>
         </article>
+        <footer css={[articleWidthStyles, footerWidthStyles]}>
```

Comment:
I think we may want to avoid using `articleWidthStyles` here; I think we probably want this to be full-width between `phablet` and `wide` instead?

![tablet](https://user-images.githubusercontent.com/53781962/89566640-6398e280-d818-11ea-83b8-8bf6a3c9c3a2.jpg)

Reply:
Takes up more space now
<img width="1012" alt="Screen Shot 2020-08-07 at 11 35 15" src="https://user-images.githubusercontent.com/11618797/89637447-182b1680-d8a2-11ea-8c9c-e49b748b16c6.png">

Reply:
Is that locally? I'm not seeing that on CODE.

Reply:
Yes that's been updated, let me push those changes